backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -102,9 +102,19 @@ public Project Get(string projectNumber)
                 .Include(p => p.Properties).ThenInclude(p => p.Parcel)
                 .Include(p => p.Properties).ThenInclude(p => p.Parcel).ThenInclude(p => p.Evaluations)
                 .Include(p => p.Properties).ThenInclude(p => p.Parcel).ThenInclude(p => p.Fiscals)
-                .Include(p => p.Properties).ThenInclude(b => b.Building)
-                .Include(p => p.Properties).ThenInclude(b => b.Building).ThenInclude(b => b.Evaluations)
-                .Include(p => p.Properties).ThenInclude(b => b.Building).ThenInclude(b => b.Fiscals)
+                .Include(p => p.Properties).ThenInclude(p => p.Parcel).ThenInclude(p => p.Classification)
```

Comment:
Not sure we have a better solution, but we're really pulling down the whole db here.

Reply:
shouldn't linq optimize this so that this is just a whole bunch of joins for one ID? we could look at the generated SQL if necessary.